### PR TITLE
Updated container name

### PR DESCRIPTION
**Pull request by ciotlosm**
- Updated container name based on the latest version just downloaded

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/hassio/debugging.markdown [hunk 1]
```diff
@@ -34,7 +34,7 @@ Make sure when you are copying the public key to the root of the SD card that yo
 journalctl -f -u resin-supervisor.service
 
 # Hass.io supervisor logs
-docker logs hassio_supervisor
+docker logs resin_supervisor
```
Comment (pvizeli) [inline source/hassio/debugging.markdown]: Can you split it to:
- resinos
- generic (like that)

Thanks a lot

Comment (ciotlosm) [inline source/hassio/debugging.markdown]: I think there might be an actual problem with the setup. I downloaded hass.io and I immediately had an update for the supervisor. I only tried to look at logs after the update and the command supplied in the docs didn't match.

**Comment by Landrash** [comment]
Any work on this PR or should it be closed?

**Comment by fabaff** [comment]
I'm closing it as there was no response from the submitter. 

Please re-open it if needed.

**Pull request closed without merging**